java-agent/instrumentation/spring-webflux-5/src/main/java8/io/opentelemetry/auto/instrumentation/springwebflux/client/TracingClientResponseSubscriber.java

```diff
@@ -18,39 +18,42 @@
   private final CoreSubscriber<? super ClientResponse> subscriber;
   private final ClientRequest clientRequest;
   private final Context context;
-  private final AtomicReference<AgentSpan> spanRef;
-  private final AgentSpan parentSpan;
+  private final AtomicReference<Span> spanRef;
+  private final Span parentSpan;
 
   public TracingClientResponseSubscriber(
       final CoreSubscriber<? super ClientResponse> subscriber,
       final ClientRequest clientRequest,
       final Context context,
-      final AgentSpan span,
-      final AgentSpan parentSpan) {
+      final Span span,
+      final Span parentSpan) {
     this.subscriber = subscriber;
     this.clientRequest = clientRequest;
     this.context = context;
     spanRef = new AtomicReference<>(span);
-    this.parentSpan = parentSpan == null ? noopSpan() : parentSpan;
+    this.parentSpan =
+        parentSpan == null
+            ? DefaultSpan.getInvalid()
+            : parentSpan; // TODO: Should we use DefaultSpan here?
```

Comment:
```suggestion
            : parentSpan;
```
I think ok to remove this TODO